packages/address/src/types.ts

```diff
@@ -0,0 +1,48 @@
+export enum FieldName {
+  FirstName = 'firstName',
+  LastName = 'lastName',
+  Country = 'country',
+  City = 'city',
+  Zip = 'zip',
+  Province = 'province',
+  Address1 = 'address1',
+  Address2 = 'address2',
+  Phone = 'phone',
+  Company = 'company',
+}
+
+export interface Address {
+  company?: string;
+  firstName?: string;
+  lastName?: string;
+  address1: string;
+  address2: string;
+  city: string;
+  // Province code
```

Comment:
do we need this comment?

Reply:
Yes, it's to specify that it's not the name of the province but the code following the [ISO_3166-2 format](https://en.wikipedia.org/wiki/ISO_3166-2).
I'll update the code mentioning this.

Reply:
I actually just realized that it's not entirely true.
I've opened an [issue on countryDB](https://github.com/Shopify/country_db/issues/169) for validation.